src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -79,12 +79,24 @@
             "org.graalvm.home.HomeFinderFeature"
     );
 
+    private static final List<String> baseNativeImageArguments = Arrays.asList(
+            "--report-unsupported-elements-at-runtime",
+            "-Djdk.internal.lambda.eagerlyInitialize=false",
+            "--no-server",
```

Comment:
not sure this is still needed or wanted, but that is probably food for another PR

Reply:
Indeed, cleanup of the arguments is work for another issue.